message.go

```diff
@@ -523,3 +528,12 @@ func (m *Message) ContentWithMoreMentionsReplaced(s *Session) (content string, e
 	})
 	return
 }
+
+// MessageInteraction contains information about the application command interaction which generated this message
+type MessageInteraction struct {
+	ID     string          `json:"id"`
+	Type   InteractionType `json:"type"`
+	Name   string          `json:"name"`
+	User   User            `json:"user"`
```

Comment:
```suggestion
	User   *User            `json:"user"`
```

Reply:
as far as I can tell the user field is always present
https://discord.com/developers/docs/interactions/receiving-and-responding#message-interaction-object-message-interaction-structure